### DISCUSSION TITLE
Livolo Dimmer switch

### DISCUSSION THREAD

**Discussion by kevincrosser** [Koenkk/zigbee2mqtt#13065] [Devices]
Hi,

I have a question. We have Livolo dimmer switches that are recognised as a switch.
We use this product (Livolo Dimmer Switch - ZigBee [https://livoloeurope.eu/livolo-zigbee-touch-light-switches/touch-livolo-smart-zigbee-dimmer-switch-lichtschakelaar/livolo-dimmer-switch-zigbee])
it is recognized as device (Zigbee switch (1, 2, 3, 4 gang))

On the Zigbee2MQTT site there is a dimmer switch supported.

Can I do something to get it working (ON/OFF Dimming)?

Hope someone can help me out. Im also just starting out using home assistant in combination with Zigbee2MQTT

**Comment by Trashsk** [comment]
Have the same issue. I just checked, manually change the zigbee module value in the database from TI0001 to TI0001-dimmer, then started Z2M. The type changed to another one, but the control also did not work. Then I switched the dimer to pairing mode and the control worked, but only while the dimmer is in pairing mode, apparently we need to ask the author to fix it, but what?

**Comment by daneZz** [comment]
I face the same issue with their new dimmers. It appears that pairing works at the zigbee2mqtt side but the dimmer still waits for a final “ok”. So it remains in pairing mode and finally times out.

**Comment by kevincrosser** [comment]
Hi,

I gave up the connection with the native zigbee dimmers. The livolo zigbee devices are not using the standard protocol. They are devices that are connecting to the hub per device. These devices are not building up a zigbee network. I now want to try using the shelly switches behind the livolo pulse switches. But these need a neutral wire.

- https://www.shelly.cloud/en-nl/products/product-overview/shelly-dimmer2
- https://livoloworld.com/livolo-touch-switch-door-bell-vl-c701b

I did not test it yet. But in theory this should work without an issue.
You can even choose between Z-Wave, ZigBee or WiFi protocol divices to connect to home assistant.
